app/src/androidTest/java/org/systers/mentorship/view/activities/LoginActivityTest.kt

```diff
@@ -1,105 +1,49 @@
-package org.systers.mentorship
+package org.systers.mentorship.view.activities
```

Comment:
can you explain the improvements you did in this test?
I see you removed a whole function, can you explain why you think we don't need this?

Reply:
@isabelcosta 
I
- removed redundant imports
- removed hardcoded `Strings` and used `getString()`
- have brought all functions that were also used in `SignUpActiivtyTest` to the utils file
- removed redundant `findEditTextInTextInputLayout(R.id.tiUsername).perform(typeText(""), closeSoftKeyboard())` as we don't need to type the empty String into the EditText (it is alredy empty)

Reply:
I actually forgot that we can bring the `findEditTextInTextInputLayout()` function to the Utils file, so I made one more commit and squashed it.

Reply:
I've updated the description :)